<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>704</width>
    <height>758</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="layoutDirection">
   <enum>Qt::LeftToRight</enum>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="styleSheet">
    <string notr="true">border-color: rgb(85, 0, 255);</string>
   </property>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>704</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QDockWidget" name="showPanelDock">
   <property name="minimumSize">
    <size>
     <width>310</width>
     <height>39</height>
    </size>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(170, 255, 255);
font: 75 10pt &quot;Times New Roman&quot;;</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <widget class="QGroupBox" name="groupBox">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>171</width>
       <height>121</height>
      </rect>
     </property>
     <property name="title">
      <string>Model Display Mode</string>
     </property>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>101</width>
        <height>91</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QRadioButton" name="PointModeBtn">
         <property name="text">
          <string>Point</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="LineModeBtn">
         <property name="text">
          <string>Line</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="FrameModeBtn">
         <property name="text">
          <string>Frame</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>140</y>
       <width>271</width>
       <height>181</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 75 10pt &quot;Times New Roman&quot;;</string>
     </property>
     <property name="title">
      <string>Roaming Setting</string>
     </property>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>174</width>
        <height>101</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QCommandLinkButton" name="AutoLBtn">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(85, 0, 255);</string>
         </property>
         <property name="text">
          <string>Auto Roaming</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCommandLinkButton" name="MannelLBtn">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(85, 0, 255);</string>
         </property>
         <property name="text">
          <string>Mannel Roaming</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QSplitter" name="splitter">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>30</y>
        <width>81</width>
        <height>60</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <widget class="QPushButton" name="NextBtn">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(170, 85, 127);</string>
       </property>
       <property name="text">
        <string>Forward</string>
       </property>
       <property name="iconSize">
        <size>
         <width>22</width>
         <height>22</height>
        </size>
       </property>
       <property name="checkable">
        <bool>false</bool>
       </property>
      </widget>
      <widget class="QPushButton" name="PreviousBtn">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(170, 85, 127);</string>
       </property>
       <property name="text">
        <string>Backward</string>
       </property>
       <property name="iconSize">
        <size>
         <width>22</width>
         <height>22</height>
        </size>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
      </widget>
     </widget>
     <widget class="QSplitter" name="splitter_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>150</y>
        <width>251</width>
        <height>23</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QPushButton" name="StartBtn">
       <property name="text">
        <string>Start</string>
       </property>
      </widget>
      <widget class="QPushButton" name="PauseBtn">
       <property name="text">
        <string>Pause</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="GlobalViewwidget" native="true">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>350</y>
       <width>291</width>
       <height>311</height>
      </rect>
     </property>
    </widget>
    <widget class="QCheckBox" name="PointPickerCbx">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>330</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Pick Point</string>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>30</y>
       <width>83</width>
       <height>91</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QCheckBox" name="LineCbx">
        <property name="text">
         <string>CenterLine</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="ModelCbx">
        <property name="text">
         <string>Model</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <zorder>layoutWidget</zorder>
    <zorder>groupBox</zorder>
    <zorder>groupBox_2</zorder>
    <zorder>GlobalViewwidget</zorder>
    <zorder>PointPickerCbx</zorder>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
